authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,615 @@
+. xref:#sts-mode-installing-running-installer[Running the installer]
+. xref:#sts-mode-installing-verifying[Verifying the installation]
+
+[NOTE]
+====
+<needinfo>
+Are they basically following the link:https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-customizations.html#installation-initializing_installing-aws-customizations[Installing a cluster on AWS with customizations] procedure here, with some specific differences for STS (and of course the addition of the preflight stuff)?
+====
+
+
+[id="sts-mode-installing-preparing"]
+=== Preparing the installer
+
+.Prerequisites
+* Obtain the pull secret for your cluster.
+
+.Procedure
+
+. Set the `$RELEASE_IMAGE` environment variable to use a sufficiently new {product-title} release:
++
+[source,terminal]
+----
+<needinfo>
+- Need information about how to do this (commands, output if applicable)
+- Is it this step? https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#retrieving-the-openshift-installer_ipi-install-installation-workflow
+- Will also need to know more specifically about "sufficiently new".
+----
+
+. Extract the AWS `CredentialsRequest` objects from the release image:
++
+[source,terminal]
+----
+$ mkdir credreqs ; oc adm release extract --cloud=aws --credentials-requests $RELEASE_IMAGE --to=./credreqs ; cat ./credreqs/*.yaml > credreqs.yaml
+----
+
+. Extract the {product-title} installer from the release image:
++
+[source,terminal]
+----
+$ oc adm release extract --command=openshift-install $RELEASE_IMAGE
+----
+
+. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config
+----
+
+. Configure the cluster to install with the CCO in manual mode:
++
+[source,terminal]
+----
+$ echo "credentialsMode: Manual" >> install-config.yaml
+----
+
+. Create install manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests
+----
+
+[id="sts-mode-installing-config-preflight"]
+=== Creating AWS resources using the sts-preflight tool
+
+When installing an {product-title} cluster that is configured to use the CCO in STS mode, you can use the `sts-preflight` tool to create the required AWS resources. This tool, which is hosted in an external repository, is provided to help configure a cluster that uses STS mode while the feature is a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+
+As an alternative, you can xref:#sts-mode-installing-config-manual[create the required AWS resources manually].
+
+.Prerequisites
+* Download the link:https://github.com/sjenning/sts-preflight[`sts-preflight` tool].
+* Ensure that you have an AWS account with sufficient permissions.
++
+<needinfo> what are these? the same permissions we get users to check in xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually Creating IAM for AWS]?
```

Comment:
We haven't compiled that precise list of permissions yet, which is why we have the vague "sufficient permissions" in the text :/

Reply:
ack